testOauth2/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Please add all files from `.vscode` folder to `.gitignore`. It's your personal VSCode settings.

Reply:
Added .gitignore file which got itself added to the repository.
I must have missed something ...